src/http-client.ts

```diff
@@ -91,13 +91,14 @@ export class HttpClient {
         status: error.status,
 
         correlationId: res.headers['inin-correlation-id'],
-        responseBody: res.text,
+        // Potentially could contain PII
+        // responseBody: res.text,
+        // requestBody: res.req._data,
+        // url: res.error.url,
+        message: 'Error making HTTP request', // res.error.message,
 
         method: res.req.method,
-        requestBody: res.req._data,
 
-        url: res.error.url,
```

Comment:
I would think url and error message should be fine.

Reply:
I had a few thoughts about the url and message: 
1. Query params _could_ contain customer data (like search params)
2. The `error.message` from `superagent` is usually `Failed to POST to ${url}` which is why I removed it (same reason as point 1)
3. This HttpClient class is exported (because the sdk uses), but it isn't publicized. I know we could keep the url since only SC and the SDK will be using it (ie. we know the URL won't contain PII), however, I am just wondering if that could open a pitfall if we ever make a change where the sdk or sc makes a request with customer supplied query params. 

Thoughts?

Reply:
that's fair. and we'd have the correlationId so we could look it up anyway.